samples/broker-trigger/README.md

```diff
@@ -1,8 +1,11 @@
-# RabbitMQ Knative Eventing Broker DLQ Example
+# RabbitMQ Knative Eventing Broker with Trigger and DLQ Example
 
 ## Prerequisites
 
-install the Broker as per [here](../../../broker/README.md)
+install the Broker running:
+`kubectl apply -f https://github.com/knative-sandbox/eventing-rabbitmq/releases/latest/download/rabbitmq-broker.yaml`
+
+or as per [here](../../broker/README.md)
 
```

Comment:
Doing `install the Broker running:` won't have the RMQ pre-requisites available, which are required. Does it make sense to link to https://github.com/knative-sandbox/eventing-rabbitmq/blob/main/broker/operator-based.md#rabbitmq-operator-based-knative-eventing-broker ?